packages/qwik/src/core/api.md

```diff
@@ -268,6 +268,9 @@ export const inlinedQrl: <T>(symbol: T, symbolName: string, lexicalScopeCapture?
 // @internal (undocumented)
 export const inlinedQrlDEV: <T = any>(symbol: T, symbolName: string, opts: QRLDev, lexicalScopeCapture?: any[]) => QRL<T>;
 
+// @alpha
+export const isInUseContext: () => boolean;

```

Comment:
i would make it `@internal` is we really need it, the fact that `_context` is defined does not mean that all use methods will work. Only useLexicalScope() works in all contexts